Framework/MDAlgorithms/test/IntegrateEllipsoidsWithSatellitesTest.h

```diff
@@ -176,12 +222,20 @@ class IntegrateEllipsoidsWithSatellitesTest : public CxxTest::TestSuite {
           dynamic_cast<const PeakShapeEllipsoid *>(&peakShape);
       auto dirs = ellipsoid->directions();
 
-      /* We have set the fake ellipsoids up to be lines along a single detectors
-       * TOF (see setup).
-       * We therefore expect the principle axis of the ellipsoid to be the same
-       * as the q-dir!
+      /* We expect an axis of the ellipsoid to be mostly
+       * along Qhkl (probably the principal axis (i.e.
+       * direction of max varaince) but not necassarily)
        */
-      TS_ASSERT_EQUALS(qDir, dirs[0]);
+      double minangle = M_PI / 2;
+      for (size_t ii = 0; ii < dirs.size(); ii++) {
+        double angle = qDir.angle(dirs[ii]);
+        if (angle > M_PI / 2) {
```

Comment:
why not use minangle here?

Reply:
It should indeed be minangle, thanks!

Reply:
The test script has been updated - it was just a type on the run number